pkgs/top-level/all-packages.nix

```diff
@@ -40157,6 +40159,10 @@ with pkgs;
     stdenv = crossLibcStdenv;
   };
 
+  serenitylibc = callPackage ../os-specific/serenity/libc.nix {
```

Comment:
Shouldn't this be serenitylibcCross, and we also have a serenitylibc package without an overridden stdenv, like every other libc?